--- cdtool-2.1.7/Makefile.in.orig	2004-08-06 14:38:17.000000000 +0200
+++ cdtool-2.1.7/Makefile.in	2004-08-07 18:27:43.345851144 +0200
@@ -47,15 +47,18 @@
 build: $(PROGRAMS)
 
 install: install-files install-links
-install-files:
-	mkdir -p $(DESTDIR)$(bindir)
-	$(INSTALL) cdctrl $(DESTDIR)$(bindir) -o root
-	$(INSTALL) cdloop $(DESTDIR)$(bindir) -o root
-	$(INSTALL) cdadd $(DESTDIR)$(bindir) -o root
-	$(INSTALL) cdown $(DESTDIR)$(bindir) -o root
-	$(INSTALL) cdtool2cddb $(DESTDIR)$(bindir) -o root
+install-dirs:
+	$(INSTALL) -d $(DESTDIR)$(bindir)
+	$(INSTALL) -d $(DESTDIR)$(libdir)
+	$(INSTALL) -d $(DESTDIR)$(mandir)/man1
+
+install-files: install-dirs
+	$(INSTALL) cdctrl $(DESTDIR)$(bindir)
+	$(INSTALL) cdloop $(DESTDIR)$(bindir)
+	$(INSTALL) cdadd $(DESTDIR)$(bindir)
+	$(INSTALL) cdown $(DESTDIR)$(bindir)
+	$(INSTALL) cdtool2cddb $(DESTDIR)$(bindir)
 
-	mkdir -p $(DESTDIR)$(mandir)/man1
 	$(INSTALL) cdctrl.1 $(DESTDIR)$(mandir)/man1
 	$(INSTALL) cdloop.1 $(DESTDIR)$(mandir)/man1
 	$(INSTALL) cdadd.1 $(DESTDIR)$(mandir)/man1
@@ -64,9 +67,9 @@
 	$(INSTALL) cdtool2cddb.1 $(DESTDIR)$(mandir)/man1
 
 	mkdir -p $(DESTDIR)$(libdir)/cdtool
-	$(INSTALL) cdtool $(DESTDIR)$(libdir)/cdtool -o root
+	$(INSTALL) cdtool $(DESTDIR)$(libdir)/cdtool
  
-LINKTARGET = ../lib/cdtool/cdtool
+LINKTARGET = $(libdir)/cdtool/cdtool
 LINKS = cdplay cdpause cdstop cdclose cdeject cdir cdinfo cdreset \
   cdvolume cdshuffle
 
@@ -74,7 +77,7 @@
 install-links-local: bindir = .
 install-links-local: install-links
 
-install-links:
+install-links: install-dirs
 	for command in $(LINKS); do \
 	  if ! test -h $(DESTDIR)$(bindir)/$$command; then \
 	    echo installing $$command; \
@@ -84,7 +87,7 @@
 	  if ! test -h $(DESTDIR)$(mandir)/man1/$$command.1; then \
 	    echo installing $$command.1; \
 	    rm -f $(DESTDIR)$(mandir)/man1/$$command.1; \
-	    ln -s cdtool.1 $(DESTDIR)$(mandir)/man1/$$command.1; \
+	    echo ".so cdtool.1" > $(DESTDIR)$(mandir)/man1/$$command.1; \
 	  fi; \
 	done
 
